history.txt

```diff
@@ -5,6 +5,46 @@
 OMERO version history
 =====================
 
+5.4.4 (March 2018)
+------------------
+
+This is a bug-fix release which also introduces some new functionality.
+
+Improvements include:
+
+-  images can now be filtered by Tag in the center panel of OMERO.web
+-  enabled search by "File" and "Tag" annotations separately in OMERO.web, as 
+   opposed to only being able to search by All annotations
+-  fixed an error when projecting an image with all the channels turned off
+-  fixed parsing of polygons and polyline ROIs so they can be opened in ImageJ
+-  fixed creation of OMERO pyramids for little endian files
+-  improved error message when login fails for OMERO.insight
+-  improved handling of idle connections in OMERO.insight
+
+Sysadmin improvements include:
+
+-  improved installation documentation for OMERO.web, and OMERO.server on
+   Debian 9, Ubuntu 16.04 and Centos7
+-  added an admin command and script to allow deletion of corrupted pyramids
+   created by a bug introduced with OMERO 5.2 (new uncorrupted pyramids can
+   then be generated)
+-  allowed enforcement of a secure connection when importing data
+
+Developer updates include:
+
+-  improved loading speed of LUT
```

Comment:
Isn't this a general improvement, not just for developers?

Reply:
I wondered similarly, I'll move it to the top list